src/main/java/com/toptrumps/core/engine/Game.java

```diff
@@ -112,6 +116,18 @@ public RoundOutcome processRoundOutcome(List<Player> winners, List<Player> playe
         return outcome;
     }
 
+    public void persistGameState(GameStateCollector gameState) {
```

Comment:
I'm refactoring the DB side of the things so that you only need to instantiate one Statistics object with a gameState and it will pass it back a set of statistics in pojo form. You want me to finish implementing this or stick to the old model?

Reply:
**INPUT CONTEXT**
Use the ```GameStateCollector``` to retrieve everything you need to know about a game. You'll use it to populate the DB through your DAO.

**OUTPUT CONTEXT**
You can create a ```Statistics``` POJO that is packed with a bunch of helper methods that facilitate the extraction of information about all past games. We'll use it to build the CLI and Online interfaces.